pkg/dochandler/didvalidator/testdata/doc.json

```diff
@@ -148,6 +148,8 @@
     {
       "id": "hub",
       "type": "IdentityHub",
+      "routingKeys": "routingKeysValue",
+      "recipientKeys": "recipientKeysValue",
```

Comment:
note: we will need a todo to adjust or remove recipientKeys when DIDComm spec advances a bit more.